.github/workflows/mintlify-broken-links.yml

```
@@ -0,0 +1,22 @@
+name: Mintlify Broken Links
+
+on:
+  push:
+    branches: [ '**' ]
```

Don't think you need this. Removing it is the same as just running it on every push.

```suggestion
    branches: [ '**' ]
```

---

haven't tried. Will do